packages/ilib-loctool-webos-javascript/test/JavaScriptFile.test.js

```diff
@@ -1081,13 +1086,26 @@ describe("javascriptfile", function() {
             type: jsft
         });
         expect(j).toBeTruthy();
-
         // should attempt to read the file and not fail
         j.extract();
 
         var set = j.getTranslationSet();
         expect(set.size()).toBe(4);
     });
+    test("JavaScriptFileTest5", function() {
+        expect.assertions(2);
+
+        var j = new JavaScriptFile({
+            project: p,
+            pathName: "./js/t5.js",
+            type: jsft
+        });
+        expect(j).toBeTruthy();
+        debugger;
+        j.extract();
+        var set = j.getTranslationSet();
+        expect(set.size()).toBe(3);
+    });
```

Comment:
Here too make another test to test that the correct contents were extracted. Preferably the string with the URL in it.

Reply:
I update test string in t5.js. Similar test strings are added t1.js as well.